hoomd/mesh.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2009-2022 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+"""Implement Mesh."""
```

Comment:
Add overview documentation, including a `See Also:` block that links to the objects that use meshes.